runtime/drivers/clickhouse/clickhouse.go

```diff
@@ -195,39 +195,20 @@ type configProperties struct {
 }
 
 func (c *configProperties) validate() error {
-	var set []string
-	if c.Host != "" {
-		set = append(set, "host")
-	}
-	if c.Port != 0 {
-		set = append(set, "port")
-	}
-	if c.Username != "" {
-		set = append(set, "username")
-	}
-	if c.Password != "" {
-		set = append(set, "password")
-	}
-	if c.Database != "" {
-		set = append(set, "database")
-	}
-	if c.SSL {
-		set = append(set, "ssl")
-	}
-
-	if c.DSN != "" {
-		if len(set) > 0 {
-			return fmt.Errorf("only one of 'dsn' or [%s] can be set", strings.Join(set, ", "))
-		}
-	}
 	if c.Managed {
-		if c.DSN != "" {
-			set = append(set, "dsn")
-		}
-		if len(set) > 0 {
-			return fmt.Errorf("managed ClickHouse does not support setting [%s] properties", strings.Join(set, ", "))
-		}
+		// In managed mode, clear all connection-related properties
+		c.DSN = ""
```

Comment:
This breaks the managed Clickhouse support. When setting `managed: true`, the provisioner provides the DSN. This code clears that DSN making it not able to connect. 

Relevant code: 
- Properties provided by the provisioner are added to the connector here: https://github.com/rilldata/rill/blob/8bec21fa48607085fae370bc2e059e9871558e2e/runtime/connection_cache.go#L137-L145
- Config keys provided by the Clickhouse provisioner: https://github.com/rilldata/rill/blob/8bec21fa48607085fae370bc2e059e9871558e2e/admin/provisioner/resources.go#L80-L84